### DISCUSSION TITLE
Focus Peaking without macro lens

### DISCUSSION THREAD

**Discussion by lmh6** [gopro/labs#1056] [Feature Requests]
i would like to ask if its possible to turn on the focus peaking without the macro lens.

that will be a great feature even for non macro uses

**Comment by dnewman-gpsw** [comment]
You can enable the macro manually, giving you access to the peaking options. However, that will slightly change the lens prescription impacting HS stability a tad.  Labs doesn't have way to turn it on without being in the macro lens mode.
